backend/backend/settings.py

```diff
@@ -85,7 +85,20 @@
 ]
 
 # For development purposes only
-CORS_ALLOW_ALL_ORIGINS = True
+CORS_ALLOW_ALL_ORIGINS = False
+
+CORS_ORIGIN_WHITELIST = [
+    'http://localhost:3000',
+]
+
```

Comment:
Local host was not working for refresh-token API, Allowed it explicitly

Reply:
yes you are right, i just searched that `CORS_ORIGIN_WHITELIST` is an alternative to `CORS_ALLOWED_ORIGINS ` and `CORS_ORIGIN_WHITELIST` has been depreceated.